db/migrations/0004_alter_skill_race.py

```diff
@@ -0,0 +1,19 @@
+# Generated by Django 4.0.2 on 2023-08-25 22:51
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('db', '0003_alter_player_guild'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='skill',
+            name='race',
+            field=models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, to='db.race'),
+        ),
+    ]

```

Comment:
Add one migration to your PR to perform all required changes, do not divide it between multiple migrations.